src/inlineExpression.tsx

```diff
@@ -137,18 +162,22 @@ export function InlineRenderer({ value }: { value?: string }): JSX.Element {
 
   // Find the expressionResult that is for this node
   const expressionMetadata = metadata?.find(p => p.expression === value);
-  const mimeBundle = expressionMetadata?.result.data as Record<string, string>;
+  const mimeBundle = expressionMetadata?.result.data as
+    | Record<string, string>
+    | undefined;
 
-  if (!mimeBundle || !expressionMetadata) {
+  if (!expressionMetadata) {
```

Comment:
@agoose77 @fperez this is actually the bug. We were early exiting if no results were found, but instead we need to actually show the error!

Reply:
I saw @fperez report today and said to myself
> I could've sworn that we used to show tracebacks

Good to see I'm not going mad (though I think this is my regression ;)). Nice find!

Reply:
Yeah, I thought I was going crazy too. Anyways, this is a slight style improvement, and will also be faster!!